packages/www/hooks/use-api.tsx

```diff
@@ -823,6 +824,75 @@ const makeContext = (state: ApiState, setState) => {
       return res;
     },
 
+    async createAsset(params): Promise<Asset> {
+      const [res, asset] = await context.fetch(`/asset/import`, {
+        method: "POST",
+        body: JSON.stringify(params),
+        headers: {
+          "content-type": "application/json",
+        },
+      });
+
+      if (res.status !== 201) {
+        throw new Error(asset.errors.join(", "));
+      }
+      return asset;
+    },
+
+    async getAssets(
+      userId: string,
+      opts?: {
+        filters?: Array<{ id: string; value: string | object }>;
+        limit?: number | string;
+        cursor?: string;
+        order?: string;
+        active?: boolean;
+        count?: boolean;
+      }
+    ): Promise<[Asset[], string, number]> {
+      const filters = opts?.filters ? JSON.stringify(opts?.filters) : undefined;
```

Comment:
Filters appear not to work on my local env, is it normal?

Reply:
it was an issue, fixed it